core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -147,34 +220,29 @@ var LearnerDashboardPage = function() {
     expect(await feedbackMessage.first().getText()).toMatch(message);
   };
 
-  this.checkIncompleteExplorationSection = async function(explorationTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteExplorationsSection();
-    await this.expectTitleOfExplorationSummaryTileToMatch(
-      explorationTitle);
-  };
+  this.navigateToCommunityLessonsAndCheckIncompleteExplorations = (
+    async function(
+        explorationTitle) {
+      await this.navigateToCommunityLessonsSection();
+      await this.expectTitleOfExplorationSummaryTileToMatch(explorationTitle);
+    });
 
-  this.checkCompleteExplorationSection = async function(explorationTitle) {
-    var completedSectionVisible = await (
-      completedExplorationsSection.isDisplayed());
-    if (!completedSectionVisible) {
-      await this.navigateToCompletedSection();
-    }
-    await this.navigateToCompletedExplorationsSection();
+  this.navigateToCommunityLessonsAndCheckCompleteExplorations = async function(
+      explorationTitle) {
+    await this.navigateToCommunityLessonsSection();
     await this.expectTitleOfExplorationSummaryTileToMatch(
       explorationTitle);
   };
 
-  this.checkIncompleteCollectionSection = async function(collectionTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteCollectionsSection();
-    await this.expectTitleOfCollectionSummaryTileToMatch(
-      collectionTitle);
+  this.navigateToCommunityLessonsAndCheckIncompleteCollections = async function(
+      collectionTitle) {
+    await this.navigateToCommunityLessonsSection();
+    await this.expectTitleOfCollectionSummaryTileToMatch(collectionTitle);
   };
 
-  this.checkCompleteCollectionSection = async function(collectionTitle) {
-    await this.navigateToCompletedSection();
-    await this.navigateToCompletedCollectionsSection();
+  this.navigateToCommunityLessonsAndCheckCompleteCollections = async function(
+      collectionTitle) {
+    await this.navigateToCommunityLessonsSection();
     await this.expectTitleOfCollectionSummaryTileToMatch(
       collectionTitle);
```

Comment:
This function looks identical to `navigateToCommunityLessonsAndCheckIncompleteCollections`